network/simple-onchain-discovery/src/builder.rs

```diff
@@ -46,7 +46,7 @@ enum State {
 
 pub struct ConfigurationChangeListenerBuilder {
```

Comment:
but what about this? why only 1 level deep :P

Reply:
Missed this one, we have so much configuration!